sops/node-specific/GDI-SOP0008_node-helpdesk-ticket-classification.md

```diff
@@ -0,0 +1,487 @@
+
+  - Tags for different files status 
+
+- **External Comms**
+
+  - Journal communications - to be used when in communication with Journals
+
+* **Security**
+
+  - EGA security issues - to be used if there is a security breach
+
+
+- **Delete**
+
+  - Non - issue - to be used when user has submitted a ticket with an issue, and then replied to the ticket saying “all ok after all” before we have had chance to reply
+
+  - Spam
+
+* **Consortium**
+
+  - Tags for different consortiums
+ 
+
+* #### Ticket priority
+
+We propose the use of three priority levels as listed below. However, each node might decide to use other levels.
+
+- **Low** - suitable for tickets that are expected to be resolved quickly and which are of routine character
+
+- **Medium** - suitable for tickets that are more complicated and might take longer time to resolve
+
+- **High** - suitable for tickets that need to be handled immediately, e.g. security breaches
+
+* #### Ticket assignee
+
+Name of node helpdesk officer assigned to the ticket.
+
+* #### Ticket status
+
+We propose the use of the model listed below. However, each node might decide to use another model.
+
+- **New** - ticket that has not yet been started to be resolved
+
+- **In progress** (or opened) - ticket in the stage of being resolved
+
+- **Resolved** - ticket that has been resolved and returned to the VHD
+
+- **Blocked** - ticket that is unable to proceed with at the moment due to e.g. external dependencies
+
+- **Deleted** - ticket that is archived e.g. since it is out of scope of the node HD
+
+##### 8.4.2 Choose tags for negative evaluation
+| Step identifier | When | Who |
+|:---|:---|:---|
+| ``4.2`` | Following the creation of the documentation issue after a negative ticket evaluation | Helpdesk officer |
+
+When the ticket was evaluated negatively then, you will choose the following tags: 
+-  **Type of ticket**: out-of-scope 
+-  **Ticket assignee**: choose from the list the name of the assigned HD member 
+-  **Ticket Status**: Deleted 
+
+#### 8.5. Follow NHD SOP for Ticket Handling 
+| Step identifier | When | Who |
+|:---|:---|:---|
+| ``5`` | After assigning tags | Helpdesk officer |
+
+The assigned HD member should refer to the designated **SOP for Ticket handling** 
+- If the ticket receives a **positive evaluation**, proceed to [SOP...].
+- If the ticket receives a **negative evaluation**, proceed to [SOP...].
+
+#### 8.6. Inform the VHD of Ticket Resolution 
+
+| Step identifier | When | Who |
+|:---|:---|:---|
+| ``6`` | After the ticket handling  | Helpdesk officer |
+
+The VHD will be informed about the ticket resolution. 
+
+When the issue has been resolved (based on a **positive ticket evaluation**), a brief notification should be sent to the VHD. You can use the following template to send an email (by the Node GDI email address) or to communicate through the node's designated system to inform the VHD.
+
+````
+Subject: Notification of Ticket Resolution: <Issue Number>
```

Comment:
Is this "Issue number" the NHD's ticket's ID, or the VHD's ticket ID? Is it always a number?